### PR TITLE
dying with the rod of asclepius now transfers your mind to the snake

### PR DESCRIPTION
**Pull request by SomeguyManperson**
it's not terribly difficult to kill or lethal so it should be fine

:cl:  
tweak: the snake created on a hippocratic oath rod user's death is now controlled by them
/:cl:

### CODE REVIEW

**Comment by jcatjmeow** [comment]
adorable tbh

**Review by jcatjmeow — APPROVED** [review]

**Comment by jcatjmeow** [comment]
wait a second

it transfers the mind to the snek
then it deletes you
wont this delete the snak who is now you?

File: code/datums/status_effects/buffs.dm [hunk 1]
```diff
@@ -413,6 +413,8 @@
 			healSnake.desc = "A mystical snake previously trapped upon the Rod of Asclepius, now freed of its burden. Unlike the average snake, its bites contain chemicals with minor healing properties."
 			new /obj/effect/decal/cleanable/ash(owner.loc)
 			new /obj/item/rod_of_asclepius(owner.loc)
+			if(owner.mind)
+				owner.mind.transfer_to(healSnake)
 			qdel(owner)
```
Comment (jcatjmeow) [inline code/datums/status_effects/buffs.dm:418]:
```suggestion
			qdel(owner)
			if(owner.mind)
				owner.mind.transfer_to(healSnake)
```

**Review by jcatjmeow — CHANGES_REQUESTED** [review]
maybe?

**Review by jcatjmeow — APPROVED** [review]
nvm this works

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**